### PR TITLE
Reskin navbar, tweak ballot, and update about page

### LINKED ISSUE #512: STAR Ballot updates
- STAR Line width (should be same as bubble width)
 - Bubbles being the same size as the STARs(ish)
 - newer update has the outside of the bubbles being black
 - make sure candidate names wrap on smaller screens
 - 
 
 
image

### LINKED ISSUE #521: Reskin navbar to be more equal vote inspired
- Black navbar
- Equal Vote logo goes to equal.vote
- Use dev.star.vote as placeholder name
- dev.star.vote also makes it into about page

### PR DESCRIPTION
**Pull request by ArendPeter**
## Description
* Navbar
   * Add Equal Vote logo
   * Change background color to black
   * Add link to help
 * About
   *  Swap out place holder title
   * Add contributor grid
 * Ballots
   * Keep ballot centered for elections you don't
   * Add star icon

### Nav bar
image
image

### About page
image
image

### Ballot
(ballot is centered with or without the side panel)
image
image
image
image
image

## Related Issues

fixes #512 
fixes #521

### CODE REVIEW

File: frontend/src/components/About.tsx [hunk 1]
```diff
@@ -1,40 +1,66 @@
-import { Button, Container, Divider, Paper, Typography } from '@mui/material'
+import { Box, Button, Container, Divider, Paper, Typography } from '@mui/material'
 import React from 'react'
 import { StyledButton } from './styles'
+import { useLocalStorage } from 'src/hooks/useLocalStorage';
```
Comment (JonBlauvelt) [inline frontend/src/components/About.tsx:4]: Why not relative path?

File: frontend/src/components/Header.js [hunk 2]
```diff
@@ -7,15 +7,17 @@ import AppBar from '@mui/material/AppBar';
...
 import MenuIcon from '@mui/icons-material/Menu';
 import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
 import Search from '@mui/icons-material/Search';
 import { useCookie } from '../hooks/useCookie'
 import { v4 } from 'uuid'
 import useAuthSession from './AuthSessionContextProvider';
 import { useThemeSelector } from '../theme';
 import useFeatureFlags from './FeatureFlagContextProvider';
+import { useLocalStorage } from 'src/hooks/useLocalStorage';
```
Comment (JonBlauvelt) [inline frontend/src/components/Header.js:20]: Same here

File: frontend/src/components/LandingPage/LandingPageHero.tsx [hunk 3]
```diff
@@ -6,11 +6,13 @@ import QuickPoll from '../ElectionForm/QuickPoll'
 import useAuthSession from '../AuthSessionContextProvider'
 import { useThemeSelector } from '../../theme'
 import useFeatureFlags from '../FeatureFlagContextProvider'
+import { useLocalStorage } from 'src/hooks/useLocalStorage'
```
Comment (JonBlauvelt) [inline frontend/src/components/LandingPage/LandingPageHero.tsx:9]: Same

**Review by JonBlauvelt — APPROVED** [review]

File: frontend/src/components/About.tsx (hunk 1 above)
Comment (ArendPeter) [reply frontend/src/components/About.tsx]: Updated! not sure why vscode chose to do absolute for the import

**Review by JonBlauvelt — APPROVED** [review]

**Pull request merged**